### ISSUE TITLE
when opening pdf, the note on the right side cannot display in dark mode

### ISSUE THREAD

**Issue by dreistein543** [ThomasFKJorna/zotero-night#12]
if we do not open any pdf files, just open a note-file on the main-interface of zotero, the note-file can display in dark mode.

<img width="971" alt="Snipaste_2022-04-27_21-22-42" src="https://user-images.githubusercontent.com/30745407/165528047-3866ec1f-53f8-4b18-b903-913746ef1bbf.png">

but if we open a pdf file, and open note-file on the right side of pdf, the note will not display in dark mode

<img width="658" alt="Snipaste_2022-04-27_21-23-50" src="https://user-images.githubusercontent.com/30745407/165528106-3dafcb6a-52c0-4f82-99aa-7341ccf4086a.png">

**Comment by ThomasFKJorna** [comment]
You're right, this something that still needs to be fixed. For now, it _should_ be darkmode if you switch from and back to the tab again though

**Comment by 7homR** [comment]
Just installed 'Night' on a different computer with Zotero, and my notes are all white (changing from one tab to another has no effect). Any other workarounds to get dark mode for the note pane on the right? Thanks.

**Comment by 7homR** [comment]
My pdf tab notes are all still white. Can we hope for a fix for this?

**Comment by AporiaYixara** [comment]
Same as 7hormR on my PC & Laptop
Also, the list of notes (Right) while viewing PDF have a light blue text on white and are unreadable (for my computers)

**Comment by 7homR** [comment]
We're quickly approaching a year since this issue was identified. Has development on this add-on stopped?

**Comment by 7homR** [comment]
While we wait for Zotero to develop [their own in-house dark mode](https://forums.zotero.org/discussion/comment/430369/#Comment_430369), a fix for this is to open the note in the main library tab, switch dark mode off and then back on, and then the note should be in dark mode.
